windows/CodePush.cs

```diff
@@ -0,0 +1,668 @@
+﻿using System;
+using System.Collections.Generic;
+using ReactNative.Bridge;
+using ReactNative.Modules.Core;
+using ReactNative.UIManager;
+using Windows.ApplicationModel;
+using Windows.Storage;
+using System.IO;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.Reflection;
+using Windows.Web.Http;
+using Windows.Storage.FileProperties;
+using ReactNative;
+
+namespace CodePush.ReactNative
+{
+    public class CodePushModule : IReactPackage
```

Comment:
I proposed this name precisely because I am familiar with React Native, and the terminology for what this class represents is a "Package". Just looking at the interface it implements makes that clear :)

Our use of the word "Package" in our API is largely a legacy issue, and something that we've started removing over time. I don't feel super strongly about this comment, but to be entirely accurate, this class doesn't represent the CodePush module, it represents the React Native package object that contributes the module to the runtime.